gui/wxpython/lmgr/layertree.py

```diff
@@ -1814,6 +1815,8 @@ def OnChangingSel(self, event):
         If the user is clicking on checkbox, selection change is vetoed.
         """
         if self.hitCheckbox:
+            # Prevent the scrollbar from scrolling up when a layer item is checked or unchecked
+            self.thisItem.GetWindow().SetFocus()
```

Comment:
I tested it and although this works, I believe this would be more readable:
```suggestion
            self.EnsureVisible(event.GetItem())
```

Reply:
Yes you are right, I have already changed it.